app/views/categories/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<section class="my-container">
+  <%= render partial:'/shared/nav_page', locals: {title: 'Spendings', backurl:
+  user_categories_path(user_id:@user.id)} %>
+  <div class="category-card-show bg-primary-light text-primary">  
+    <div class="category-card-wrapper">
+      <i class="material-icons outlined material-icon"><%= @category.icon%></i>
+      <h4 class="category-card-title text-primary-light"><%= @category.name%></h4>
+      <h4 class="category-card-amount text-primary">$ <%= @category.total_amount %></h4>
+    </div>
+  </div>
+  <div class="spendings-container">
+    <% @spendings.each do |spending| %>
+    <div class="spending-card bg-primary-light">
+      <h4 class="show-spendings-title text-primary-light">
+        <%= spending.name %>
+      </h4>
+      <h4 class="show-spendings-amount text-primary"> Amount: $ 
+        <%= spending.amount %>
+      </h4>
+    </div>
+    <% end %>
```

Comment:
- [ ] Let us style the individual spending cards so that they look like actual cards and also like the original design.
- [ ] Please use the recommended font-family and font-weight
### Design Guideline
![image](https://github.com/roniy68/budget-app/assets/46839548/d88aa78a-766f-4474-8071-af04c5fa687f)

### Your Implementation
![image](https://github.com/roniy68/budget-app/assets/46839548/5daa80d1-4c0e-45cb-b390-1ad3fdc7921c)

Reply:
👍 Resolved 
![Uploading image.png…]()